proxy/src/main/java/com/wavefront/agent/listeners/otlp/OtlpProtobufUtils.java

```diff
@@ -100,20 +103,22 @@ public static void exportToWavefront(ExportTraceServiceRequest request,
     }
 
     for (Pair<Span, SpanLogs> spanAndLogs : fromOtlpRequest(request, preprocessor, defaultSource)) {
-      if (wasFilteredByPreprocessor(spanAndLogs._1, spanHandler, preprocessor)) {
-        continue;
-      }
+      Span span = spanAndLogs._1;
+      SpanLogs spanLogs = spanAndLogs._2;
```

Comment:
Can we use another "structure", Pair with its `_1` and `_2` makes the code ugly and hard to read.

Reply:
I agree - I was looking for something lightweight to group "related" things together. 

We could switch to Apache Commons' [Pair](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/tuple/Pair.html), which would look like:
```java
Span span = spanAndLogs.left();
SpanLogs spanLogs = spanAndLogs.right();
```

Or, use JDK's `AbstractMap.SimpleEntry<Span, SpanLogs>`:
```java
Span span = spanAndLogs.getKey();
SpanLogs spanLogs = spanAndLogs.getValue();
```

Are either of those preferable? If not, any other ideas? I use this same pattern elsewhere in `OtlpProtobufUtils.java`, such as [grouping counters with the associated object](https://github.com/oppegard/wavefront-proxy/pull/11/files#diff-0b958590de9d5f133cce391332fd6ac139feb19a89ef88bf938180e1e303c12cR94-R95) that determines whether or not to increment the counter.

Reply:
May we can create a simple class with two members.

Reply:
Ok, I can create a simple container class to hold the two members.